src/AppBundle/Resources/es6/app.js

```diff
@@ -7,6 +7,72 @@ import $ from 'jquery';
 import './ngsite';
 import '../sass/style.scss';
 
-$(() => {
-  // Put your code here
+import HeaderNav from './components/HeaderNav.component';
```

Comment:
Shouldn't all this be in ngsite.js? Or am I missing something here?

Reply:
It should, yes. `app.js` is intended to hold only custom project code.